{
    "repository": "mido/mido",
    "clone_url": "https://github.com/mido/mido.git",
    "timestamp": "2023-05-29T14:52:35.780055Z",
    "clone_success": true,
    "number of actions": 1,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_stdout": "[Test/build] \ud83d\ude80  Start image=crawlergpt:latest\n[Test/build]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[Test/build]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Test/build]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[Test/build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1000 /tmp/de65f406-fe28-11ed-a890-af2cc187fc11/mido-mido] user=0 workdir=\n[Test/build]   \u2601  git clone 'https://github.com/actions/setup-python' # ref=v4\n[Test/build] Unable to reset to bd6b4b6205c4dbad673328db7b31b7fab9e241c0: unexpected EOF\n[Test/build] \ud83c\udfc1  Job succeeded\n",
    "actions_stderr": "Error: unexpected EOF\n"
}